pkg/api/v1/types.go

```diff
@@ -500,6 +504,63 @@ const (
 	ClaimBound PersistentVolumeClaimPhase = "Bound"
 )
 
+// PersistentVolumeConditionType is a valid value for PersistentVolumeCondition.Type
+type PersistentVolumeConditionType string
+
+// These are valid conditions of PersistentVolume.
+const (
+	// PersistentVolumeBound means the volume is bound to a claim for storage and can be
+	// consumed as a storage resource by the claimant
+	PersistentVolumeBound PersistentVolumeConditionType = "Bound"
+	// PersistentVolumeRecycled is the recycled state of a PersistentVolume per the volume's reclaim policy
+	PersistentVolumeRecycled PersistentVolumeConditionType = "Recycled"
+	// PersistentVolumeRecycled is the provisioned state of a PersistentVolume
+	PersistentVolumeProvisioned PersistentVolumeConditionType = "Provisioned"
+)
+
+// PersistentVolumeCondition contains the details for the current condition of this volume
+type PersistentVolumeCondition struct {
+	// Type is the type of the condition.
```

Comment:
Don't refer to Go field names in API field comments. Either don't mention the field name or refer to the json field name.